.github/ISSUE_TEMPLATE/new_release.md

```diff
@@ -6,37 +6,81 @@ labels: "kind/release"
-
-### Tasks
-- [ ] Go / No Go [hackmd doc](https://hackmd.io/team/flatcar?nav=overview) created from previous one for test tracking and release notes: [link](TODO) ()
-- [ ] Check that the kernel PR are merged for the channel(s) to be released. ()
-- [ ] All planned changes are merged & blocker bugs are fixed, last nightly reviewed for problems ()
-- [ ] Branch off main (only new major) ()
-- [ ] Tag build branch ()
-- [ ] Start Jenkins builds, post links as comment ()
-- [ ] Test status captured in hackmd doc ()
-- [ ] All tests are green ()
-- [ ] Release notes / announcements added to Go/No Go doc from `container/image_changes` job by selecting `Timestamps: None` ()
-- [ ] QA: check diff for image file list, packages, and image size in the image job output ()
-- [ ] Matrix Go/No Go meeting (inc. release notes review) ()
-- [ ] Update payload signed/uploaded ()
-- [ ] `scripts` release created on GitHub ()
-- [ ] Release job `container/release` ran ()
-- [ ] Images uploaded with `copy-to-origin.sh` ()
-- [ ] Azure offer updated & publishing started ()
-- [ ] Azure Go Live ()
-- [ ] GCP Offer deployment package uploaded to Google bucket and submitted for verification ()
-- [ ] GCP Offer submit for verification ()
-- [ ] GCP Offer Publish ()
-- [ ] CAPI for new Stable:
+The release of the following Flatcar Container Linux version is planning <MONTH> <DAY>, <YEAR>.
+
+- Alpha <VERSION>
+- Beta <VERSION>
+- Stable <VERSION>
+- LTS <VERSION>
+
+## 1. Preparation
+
+Owner:
+
+- [ ] A. Go/No-Go [hackmd doc](https://hackmd.io/team/flatcar) created from HackMD template for test tracking and release notes: (update the link here)
+- [ ] B. All planned changes are merged & blocker bugs are fixed, last nightly reviewed for problems.
+- [ ] C. Check that the kernel PR are merged for the channel(s) to be released.
+
+## 2. Build
+
+Owner:
+
+- [ ] A. Branched off the `main` branch (only for new major).
+- [ ] B. Tagged all the build branches.
+- [ ] C. Started the Jenkins builds, post the links into this issue as comment.
+
+
+## 3. Release
+
+### i. The Go/No-Go Meeting
+
+Owner:
+
+- [ ] A. Test status captured in the HackMD document.
+- [ ] B. All tests are Green.
+- [ ] C. Release notes / announcements added to Go/No Go doc from `container/image_changes` job by selecting `Timestamps: None`.
+- [ ] D. QA: Checked the diff for image file list, packages, and image size in the `container/image_changes` job output.
+- [ ] E. Conduct Matrix Go/No Go meeting (inc. release notes review).
+
+
+### ii. Artifacts goes public
+
+Owner:
+
+- [ ] A. Created the Release Notes on the `scripts` repo for the respective tags on GitHub as GitHub Releases.
+- [ ] B. Ran the `container/release` Release job.
+- [ ] C. Images uploaded with `copy-to-origin.sh`.
+- [ ] D. Symlink to "current" updated with `set-symlink.sh`.
+- [ ] E. Website updated with `./update-flatcar-versions.sh`.
```

Comment:
You would probably mean "Website PR created with `./update-flatcar-versions.sh` and requested review."?